src/util.js

```diff
@@ -39,6 +39,15 @@ Util.isIOS = (function() {
   };
 })();
 
+Util.isWebViewAndroid = (function() {
+  var isWebViewAndroid = navigator.userAgent.indexOf('Version') !== -1 &&
+      navigator.userAgent.indexOf('Android') !== -1 &&
+      navigator.userAgent.indexOf('Chrome') !== -1;
```

Comment:
this would fail for a page embedded as [a Chromium-based WebView]( https://github.com/uupaa/UserAgent.js/blob/5d90111/test/testcase.js#L178-L190), as well as [these several cases](https://github.com/uupaa/UserAgent.js/blob/5d90111/test/testcase.js#L403-L451).